Tasks/PWGHF/HFTrackIndexSkimsCreator.cxx

```diff
@@ -539,16 +539,17 @@ struct HfTrackIndexSkimsCreator {
   }
 
   /// Method to perform selections for 2-prong candidates before vertex reconstruction
-  /// \param hfTracks is the array of 2 tracks
+  /// \param hfTrack0 is the first daughter track
+  /// \param hfTrack1 is the second daughter track
   /// \param massHypos is a 2D array containing the mass hypotheses for the 2-prong channels
   /// \param cutStatus is a 2D array with outcome of each selection (filled only in debug mode)
```

Comment:
Can you please fix the documentation as well?
`massHypos` seems confused with `whichHypo`

Reply:
Done!